sklearn/utils/tests/test_graph.py

```diff
@@ -5,8 +5,10 @@
 from scipy import sparse
 
 from sklearn.utils.graph import graph_laplacian
+from sklearn.utils.testing import ignore_warnings
 
 
+@ignore_warnings(category=DeprecationWarning)
 def test_graph_laplacian():
```

Comment:
to me, the point of the test currently is simply to check that importing `sklearn.utils.graph_laplacian` is not broken, because we are still supporting it for two releases.  When that import point will go away, the test will fail, reminding us to remove it. I could replace it by a simpler guard test that just imports, checks for deprecation warning, and returns, but this is lazier.

Actual functionality tests for math properties should from now on go into scipy, IMO, unless we find a bug and backport it again.